webpack.config.js

```diff
@@ -31,7 +31,9 @@ const extractConfig = {
 
 const entryPointNames = ["config-app", "blocks"];
```

Comment:
We could remove even more if we would come up with a way to expose `wp-scripts build` which makes it possible to provide entry points :)

Reply:
I suspect this will be hard to make generic enough for all use-cases, but might be good to provide at least a no-config option for simple plugins

Reply:
As long as `webpack.config.js` is JS file you can tweak it however you want. So working default should work and everyone else can extend and adjust 👍